source/user/pbc/gto.rst

```diff
@@ -0,0 +1,167 @@
+.. _user_pbc_gto:
+
+Crystal structure
+*****************
+
+*Modules*: :mod:`pbc.gto`
+
+Initializing a crystal
+======================
+
+Initializing a crystal unit cell is very similar to the initialization
+of a molecule. Instead of the :class:`gto.Mole` class, 
+one uses the :class:`pbc.gto.Cell` class to define a cell::
+
+  >>> from pyscf.pbc import gto
+  >>> cell = gto.Cell()
+  >>> cell.atom = '''H  0 0 0; H 1 1 1'''
+  >>> cell.basis = 'gth-dzvp'
+  >>> cell.pseudo = 'gth-pade'
+  >>> cell.a = numpy.eye(3) * 2
+  >>> cell.build()
+
+The other two equivalent ways to initialize a molecule 
+introduced in :numref:`user_gto` also apply here::
+
+  >>> from pyscf.pbc import gto
+  >>> cell = gto.Cell()
+  >>> cell.build(
+  ...   atom = '''H  0 0 0; H 1 1 1''',
+  ...   basis = 'gth-dzvp',
+  ...   pseudo = 'gth-pade',
+  ...   a = numpy.eye(3) * 2)
+
+  >>> import pyscf
+  >>> cell = pyscf.M(
+  ...   atom = '''H  0 0 0; H 1 1 1''',
+  ...   basis = 'gth-dzvp',
+  ...   pseudo = 'gth-pade',
+  ...   a = numpy.eye(3) * 2)
+
+  >>> from pyscf.pbc import gto
+  >>> cell = gto.M(
+  ...   atom = '''H  0 0 0; H 1 1 1''',
+  ...   basis = 'gth-dzvp',
+  ...   pseudo = 'gth-pade',
+  ...   a = numpy.eye(3) * 2)
+
+
+Lattice vectors
+---------------
+
+The crystal initialization requires an additional parameter :attr:`~Cell.a`, which
+represents the lattice vectors. The format of :attr:`~Cell.a` is array-like::
+    
+    cell.a = numpy.eye(3) * 2
+    cell.a = [[2,0,0],[0,2,0],[0,0,2]]
+
+Each row of the 3-by-3 matrix of :attr:`~Cell.a` represents a lattice vector
+in Cartesian coordinate, with the same unit as the input :attr:`atom` parameter.
+
+.. note:: It is recommended that the input lattice vectors (row by row) form a 
+          right-handed coordinate system.
+
+Basis set and pseudopotential
+-----------------------------
+
+PySCF uses the crystalline Gaussian-type orbitals as the basis functions
+for solid calculations. The predefined such basis sets include 
+the valence basis sets that are optimized for the GTH pseudopotentials 
+(a whole list can be found in :source:`pyscf/pbc/gto/basis` and :source:`pyscf/pbc/gto/pseudo`).
+The input format of `basis sets`_ for the :class:`Cell` object is the same 
+as that for the :class:`Mole` object.
+In addition, the predefined basis sets and ECPs for molecular calculations 
+can be used in solid calculations as well
```

Comment:
I'm wondering whether the molecular section should talk about ECPs...

Does PySCF implement GTH pseudopotentials, or how are the GTH sets used in PySCF?

Reply:
I will clarify in next PR